tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_friendship.c

```diff
@@ -815,6 +815,26 @@ static void test_other_group(void)
 	PASS();
 }
 
+/** LPN terminate cb test.
+ *
+ * Check that terminate cb is not triggered when there is no established
+ * connection.
+ */
+static void test_lpn_term_cb_check(void)
+{
+	bt_mesh_test_setup();
+
+	bt_mesh_lpn_set(true);
+	k_sleep(K_MSEC(400));
+	bt_mesh_lpn_set(false);
+
+	if (!evt_wait(LPN_TERMINATED, K_SECONDS(30))) {
```

Comment:
Is it necessary to wait 30 seconds? Could the wait time be shorter (5 or 10 sec), or this test will take more than 30 sec to execute.

Reply:
No, I guess it could be shorter. 30 Sec seems to be a commoly used period. Since it is simulated time I guess that the impact is minimal regardless?